src/thoughts-on/escaping-the-labyrinth.html

```diff
@@ -0,0 +1,610 @@
+    <meta
+      name="description"
+      content="How to navigate an interaction with an LLM without getting lost"
+    />
+    <link rel="stylesheet" href="/src/styles/common.css" />
+    <title>Escaping the labyrinth</title>
+    <link rel="icon" type="image/x-icon" href="/src/media/favicon.ico" />
+  </head>
+  <body class="body">
+    <header>
+      <a href="#main" class="skip">Skip to main content</a>
+      <div>
+        <h1>Escaping the labyrinth</h1>
+      </div>
+    </header>
+    <main id="main" class="main">
+      <div class="byline">
+        <img
+          class="byline-logo"
+          src="/src/media/pac-logo.svg"
+          alt="People and code logo."
+        />
+        <div>
+          <div class="byline-brand">People and Code</div>
+          <div class="byline-tagline">At Your Disposal</div>
+        </div>
+      </div>
+      <p>
+        My prompts to LLMs for input sometimes lead to a stream of code and
+        explanatory notes that I don't want. It's tempting to try and fix things
+        in media res but I usually start over. This affects me in a couple of
+        ways: I feel bad that I've wasted resources through my carelessness; the
+        mess I've created takes the edge off my enthusiasm.
+      </p>
+      <p>
+        This typically happens when I'm working on a small project; something
+        that might require a few files, and a handful of functions. And disaster
+        can strike at any moment. I ask for tests but forget to mention that I'm
+        using vitest now and not jest, or I fail to stipulate that I want to be
+        asked before adding dependencies. Myriad unread lines spool down the
+        screen.
+      </p>
+      <p>Then I remembered the ball of thread Ariadne gave to Theseus.</p>
+      <aside>
+        <h2 id="ariadnes-thread">Ariadne's Thread</h2>
+        <p>
+          In Greek mythology, the Cretan princess, Ariadne handed Theseus a ball
+          of thread before he entered the labyrinth at Knossos so that he might
+          find his way back to safety after confronting, and killing, the
+          Minotaur.
+        </p>
+      </aside>
+      <section aria-labelledby="the-thread">
+        <h3 id="the-thread">The thread</h3>
+        <p>
+          <em
+            ><strong>Thread</strong>: The record of an interaction between a
+            developer and an LLM. Contains the thread prompt used to initialise
+            the interaction and a collection of snapshots.</em
+          >
+        </p>
+        <p>
+          I will use a <strong>thread</strong> to encapsulate the interactions I
+          have with LLMs.
+        </p>
+        <p>
+          My thread is a simply structured record written in JSON. Here's the
+          first part of it:
+        </p>
+        <aside>
+          <section aria-labelledby="the-json-thread">
+            <h2 id="the-json-thread"><em>thread.json</em></h2>
+            <pre id="json-thread" class="json">
```

Comment:
Why not work with the RSS feed generator example directly? It would make the examples more practical. If the users want the general structure, they can refer to the repo.

Reply:
Good idea.